doozer

```diff
@@ -1483,13 +1484,80 @@ def release_mirror(runtime, src_dest, image_stream, is_base, pattern):
 
     yellow_prefix("Images skipped due to invalid naming:\n")
     for img in sorted(invalid_name_items):
-	click.echo(" {}".format(img))
-	click.echo()
+        click.echo(" {}".format(img))
+        click.echo()
 
     yellow_prefix("Images skipped due to missing source:\n")
     for img in sorted(missing_source_items):
-	click.echo(" {}".format(img))
-	click.echo()
+        click.echo(" {}".format(img))
+        click.echo()
+
+
+@cli.command("beta:reposync", short_help="Generate .repo file from group data")
+@click.option("-o", "--output", help="Output directory to sync to", required=True)
+@click.option("--repo-type", metavar="REPO_TYPE", envvar="OIT_IMAGES_REPO_TYPE",
+              default="unsigned",
+              help="Repo group type to use for repo file generation (e.g. signed, unsigned).")
+@pass_runtime
+def beta_reposync(runtime, output, repo_type):
+    """Generate a yum.conf compatible file from group data
+    All repos will be disabled in the file.
+
+    See `doozer --help` for more.
+
+    Examples:
+
+    Write unsigned.repo:
+
+        $ doozer --group=openshift-4.0 beta:reposync -o yum.conf --repo-type unsigned
+
+    """
+    runtime.initialize(clone_distgits=False)
+    repos = runtime.repos
+
+    yum_conf = """
+[main]
+cachedir=/tmp/cache/yum/$basearch/$releasever
+keepcache=0
+debuglevel=2
+logfile=/tmp/yum.log
+exactarch=1
+obsoletes=1
+gpgcheck=1
```

Comment:
Consider `gpgcheck=0` to avoid surprises. i.e. sync what is there, regardless of whether it is signed.